.github/workflows/_test_rosetta.yaml

```diff
@@ -0,0 +1,42 @@
+name: ~test and lint Rosetta
+
+on:
+  # Manual trigger
+  workflow_dispatch:
+  # Called from another workflow
+  workflow_call:
+    inputs:
+      ROSETTA_IMAGE_TAG:
```

Comment:
I recently decided to use the complete image address rather than only the tag part for the workflow input. See [for example here](https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/workflows/_test_jax.yaml). This will make it easier for us to switch between the public facing `nvidia/t5x` and internal `nvidia/jax-toolbox-internal` repos. Could you please update this input argument as in the `_test_jax.yaml` example?

Reply:
makes sense. done